src/components/intro-banner.js

```diff
@@ -29,13 +29,13 @@ const IntroBanner = () => {
           {intl.formatMessage(
             { id: "intro-description" },
             {
-              cityBureauLink: (
+              siteOwnerLink: (
                 <a
                   href="https://www.citybureau.org/"
```

Comment:
This href is a trivial example, but there is kind of an open question about how, if there is a version of the project to be used as a template, it would probably be better to avoid committing these kinds of hard-coded links

Less trivial is like how I accidentally sent ya'll a form submission when I was playing with the project :-D